vowpalwabbit/reduction_stack.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "reductions_fwd.h"
+
+struct vw;
+
+typedef VW::LEARNER::base_learner* (*reduction_setup_fn)(VW::setup_base_fn&, VW::config::options_i&, vw&);
+
+namespace VW
+{
+struct status_quo : public setup_base_fn
```

Comment:
`status_quo` -> `default_reduction_stack_setup` or something more descriptive, `status_quo` will loose it's meaning in a couple of months

Reply:
done